### ISSUE TITLE
Save filter settings

### ISSUE THREAD

**Issue by argv-minus-one** [neotron/EDPathFinder#13]
The filter settings, like “included settlement sizes” and “maximum number of hops in route”, are reset when closing and restarting the tool. Please consider saving these settings instead.

Version 1.1.2.

**Comment by neotron** [comment]
Good idea. Currently don't have any persistent settings, but could be done for a future release.
